### ISSUE TITLE
Desing/UX

### ISSUE THREAD

**Issue by gaboesquivel** [eoscostarica/dmeetup#5]
We need to start working on the UX workflows and image of the dApp. 
Including the logo :)

**Comment by murillojorge** [comment]
I'll prepare a basic workflow so we can discuss it.

**Comment by murillojorge** [comment]
I’d like to propose a 5 stage design workflow. 
Stage 1 - Define specs and requirements.
Stage 2 - Research market and personas.
Stage 3 - Set goals and tasks.
Stage 4 - Low fidelity prototyping.
Stage 5 - High fidelity prototyping.

Where can we publish the documentation generated from this workflow?

**Comment by gaboesquivel** [comment]
Do your thing @murillojorge @dianitica.  
There's one particular aspect of this project that one needs to be aware of, it's not going to be so linear, many things are going to happen in parallel and that's one of the exciting challenges. Specifically, low fidelity prototyping has already started, we have a first simple page and were working on a #33 and other contributors will send PRs. This is fun.

So we need to keep track of what going on the low fidelity proto and have a 2 way feedback cycle, I love this article https://rauchg.com/2015/pure-ui. I think we are doing an asynchronous pure ui development :).

Bottom line, I agree. However consider development is already doing stuff while you work on Design/UX, so we really need a two way feedback cycle where we collaborate in parallel. Hopefully I explained it well, no so linear. While this workflow is followed by Design/UX other things are happening in development :) .. we need two way feedback. Gillermo's Pure UI post explains nicely I think

**Comment by gaboesquivel** [comment]
Let's try to keep as much as possible here. 

There's already a an issue for personas #39, we can create another one ( or more ) for the UI specs and requirements. There one for the MockedUI #37 which is a quick and dirty functional demo with minimum feats. One for mocked voting component #33.

**Comment by murillojorge** [comment]
Ok, not a problem, we'll couple the UX workflow to the ongoing work as needed.
Data models as guides for what needs to be represented and how, it's a very interesting idea indeed.

**Comment by gaboesquivel** [comment]
@murillojorge this is your realm.  I think we can close this one, it's too broad.
